src/components/DRB/DRBContainer.tsx

```diff
@@ -33,8 +33,8 @@ const DRBContainer = ({
   const drbQuery = getDRBQueryStringFromSearchParams(searchParams)
 
   return (
-    <Card isBordered>
-      <CardHeading level="h3">
+    <Card backgroundColor="ui.bg.default" p="s" borderRadius="5px">
+      <CardHeading level="h6">
```

Comment:
I think this should be an h2 to be on the same level as the "Displaying 1-50 of ...." text. The VQA doc says it's in the _style_ of h6 so do `size="h6"` instead.